native/cocos/renderer/pipeline/GlobalDescriptorSetManager.cpp

```diff
@@ -61,6 +61,11 @@ void GlobalDSManager::activate(gfx::Device *device, RenderPipeline *pipeline) {
         gfx::Address::CLAMP,
     });
 
+    uint maxJoints = (_device->getCapabilities().maxVertexUniformVectors - 38) / 3;
+    maxJoints = maxJoints < 256 ? maxJoints : 256;
+    SkinningJointCapacity::jointUniformCapacity = maxJoints;
+    UBOSkinning::initLayout(maxJoints);
```

Comment:
I think it's ok the initialized UBOSkinning layout here, please use the same approach in TypeScript engine